webapp/channels/src/components/profile_popover/components/profile_popover_call_button_wrapper/index.tsx

```diff
@@ -109,29 +95,25 @@ const CallButton = ({
         id: 'webapp.mattermost.feature.start_call',
         defaultMessage: 'Start Call',
     });
-    const iconButtonClassname = classNames('btn icon-btn', {'icon-btn-disabled': disabled});
```

Comment:
Does this not need a separate CSS class any more for when its disabled? I'm not sure why it uses `aria-disabled` below instead of just `disabled`, but that's an existing thing

Reply:
Yes its now using the standard button icons. The reason it has styling like this instead of having attribute a disabled is may be because this button is being passed down in the userCall widget which triggers the handler on it

Reply:
That sounds a bit weird to me, but it's already pretty weird that this component is part of the web app to begin with so we don't need to fix all that right now anyway